File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PackagesUpdateExternalidRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PackagesUpdateExternalidRequestItem(
    @JsonProperty("ExternalId")
    val externalId: String? = null,
    @JsonProperty("PackageLabel")
    val packageLabel: String? = null
)

File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/PackagesUpdateExternalidRequestItem.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class PackagesUpdateExternalidRequestItem
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("ExternalId")]
        public string? ExternalId { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("PackageLabel")]
        public string? PackageLabel { get; set; }
    }
}
